lib/Event/SmsMessageSent.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Fazland\SkebbyRestClient\Event;
+
+use Fazland\SkebbyRestClient\DataStructure\Sms;
+use Symfony\Contracts\EventDispatcher\Event;
+
+final class SmsMessageSent extends Event
```

Comment:
As per previous comment, I think it's better to require framework-agnostic `psr/event-dispatcher`: `Event` reference here should be removed accordingly.

Question: should this event be stoppable? I don't think that this can be stopped.

Reply:
So our event would not extend anything?
I'm not sure that this could be usable with Symfony...
By the way, event-dispatcher-contracts is indeed framework-agnostic, since it doesn't depend on any other symfony component

Reply:
I double-checked and it looks like, actually, Symfony doesn't require an event to extends any of its classes.